config/cms.php

```diff
@@ -93,6 +93,17 @@
 
     'backendSkin' => \Backend\Skins\Standard::class,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Back-end post sign-out redirect URL.
+    |--------------------------------------------------------------------------
+    |
+    | This acts as the default URL (relative or absolute) where a back-end user
+    | will be redirected to after she or he signs out.
+    */
+
+    'postSignoutRedirectUrl' => 'backend',
```

Comment:
This would be better located in the auth.php under the `auth.backend.redirectAfterLogout` config key; otherwise looks good!

Reply:
And actually, it might be better to implement the auth event system changes from https://github.com/wintercms/winter/issues/296 instead and then rather than a config value we would just provide an event that you could listen to in order to do any sort of custom processing you desire, including redirecting.

Reply:
Event based approach is no doubt a better option! The only advantage of the PR is that it solves the issue here and now.

Reply:
In the interest of meeting both our goals (yours a solution now, mine a solution that I won't have to revert and replace later), how about we just implement the `backend.user.logout` event for now here and make it so that if a response is returned then it returns the response otherwise it does the default redirect action?